src/delta.rs

```diff
@@ -382,6 +409,28 @@ fn prepare(line: &str, append_newline: bool, config: &Config) -> String {
     }
 }
 
+fn _line_number_style(line_number: Option<usize>) -> String {
+    match line_number {
+        Some(x) => format!("{:^4}", x),
+        None => format!("    "),
```

Comment:
does this handle line numbers over 4 digits without breaking alignment? Maybe if it were deferred to `paint_lines` as suggested below, it would be easier to ensure that longer numbers are handled since you'd have access to all of them.

Reply:
👍 deferred to `paint_lines`. Need to take a look at how it handles >4 digits.